pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1801,7 +1801,7 @@
       "BaselineVersion": "4.0.0",
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.0.0",
-        "4.0.1.0": "4.3.0"
+        "4.1.0.0": "4.3.0"
```

Comment:
Orphaned entries will get cleaned up once I merge some build tools changes that are coming.

As for 4.3.0 vs 4.4.0 I'll be making that change shortly and will update the index at that point.

For the most part you don't need to worry too much about the index other then when the packaging tells you to update it.